contour/src/main/kotlin/com/squareup/contour/constraints/SizeConfigSmartLambdas.kt

```diff
@@ -0,0 +1,35 @@
+package com.squareup.contour.constraints
+
+import android.view.View
+import androidx.core.view.children
+import com.squareup.contour.ContourLayout
+import com.squareup.contour.constraints.SizeConfigSmartLambdas.Orientation.HORIZONTAL
+import com.squareup.contour.constraints.SizeConfigSmartLambdas.Orientation.VERTICAL
+import kotlin.math.max
+
+internal object SizeConfigSmartLambdas {
+  fun matchParent(): SizeConfigLambda = { it }
+  fun wrapContent(view: ContourLayout, orientation: Orientation): SizeConfigLambda = {
+    view.run {
+      children
+          .filter { it.visibility != View.GONE }
```

Comment:
if you're using ktx, do you wanna use `it.isVisible` as well?

Reply:
sgtm!

Reply:
I'm going to remove ktx since we're only using one function